hls4ml/templates/vivado/nnet_utils/nnet_pooling_stream.h

```diff
@@ -2,8 +2,10 @@
 #define NNET_POOLING_STREAM_H_
 
 #include "utils/x_hls_utils.h"
+#include "ap_shift_reg.h"
```

Comment:
Might not work for Python API if `ap_shift_reg.h` is not included in vivado template.

Reply:
Added the ap_shift_reg.h header file from Vivado HLS library to ap_types. The Python API should now work with the compile() function without throwing an error.